src/pages/lessons/Lessons.tsx

```diff
@@ -378,6 +382,7 @@ const Lessons = ({
         path={`${match.url}/:category/:subcategory/:lessonPath`}
         render={(props) => (
           <Lesson
+            key={props.location.pathname}
```

Comment:
What prompted adding this? I'm just curious

Reply:
oops, that's a mistake. I tested something here. I'm going to remove this